### PR TITLE
fix(Box): correct bugs with Box

### PR DESCRIPTION
**Pull request by CITguy**
- Fix CSS for `div.hxBox` demo
- Add HXDivElement back to list of all elements

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit 740b752833774ba3cda4a8e1705c628bc97d83dc

https://app.netlify.com/sites/helix-ui/deploys/5b7c531373f2cf35546c101e

File: docs/styles/demo/box-docs.less [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .box-demo {
+  .hxBox,
```
Comment (CITguy) [inline docs/styles/demo/box-docs.less]: Before | After
----- | -----
screen shot 2018-08-21 at 12 59 55 pm | screen shot 2018-08-21 at 1 00 38 pm

File: src/helix-ui/elements/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export { HXAlertElement } from './HXAlertElement';
 export { HXBusyElement } from './HXBusyElement';
 export { HXCheckboxElement } from './HXCheckboxElement';
 export { HXDisclosureElement } from './HXDisclosureElement';
+export { HXDivElement } from './HXDivElement';
```
Comment (CITguy) [inline src/helix-ui/elements/index.js]: This might have accidentally gotten dropped as a result of a rebase involving #309

**Comment by 100stacks** [comment]
Dev LGTM

**Pull request merged**